share/lib/python/scripts/_binwrapper.py

```diff
@@ -41,6 +43,28 @@ def _config_exe(exe_name):
     return os.path.join(NRN_PREFIX, 'bin', exe_name)
 
 
+def _wrap_executable(output_name):
+    """Create a wrapper for an executable in same dir. Requires renaming the original file.
+    Executables are typically found under arch_name
+    """
+    print("Wrapping special binary...")
```

Comment:
pure nitpick - this is just extra message for user IMO....

Reply:
Yup, too verbose for the purpose at hand. Let;s remove it.